### DISCUSSION TITLE
Nominating @arjun024 for the role of Maintainer

### DISCUSSION THREAD

**Discussion by thitch97** [paketo-buildpacks/feedback#233] [Python Team]
I'd like to nominate @arjun024 as a maintainer for the Python subteam. He has been a long-time contributor to the Python buildpacks as a former maintainer and has made large contributions of his own to the space (e.g. [cpython](https://github.com/paketo-buildpacks/cpython/commits?author=arjun024) & [pip](https://github.com/paketo-buildpacks/pip/commits?author=arjun024)).

👍 / 👎 ?

cc/ @paketo-buildpacks/python-maintainers
@paketo-buildpacks/steering-committee

**Comment by ForestEckhardt** [comment]
Congrats @arjun024! You have been added to the Python Maintainer team.

Please open a PR on the [TEAMS.md](https://github.com/paketo-buildpacks/community/blob/main/TEAMS.md) to add @arjun024 to the list of maintainers.
